# This represents a the wheel vector.
# [front left, rear left, rear right, front right] wheel.

float64 fl 					# front left wheel
float64 rl 					# rear left wheel
float64 rr 					# rear right wheel
float64 fr 					# front right wheel
